inspire_dojson/cds/rules.py

```diff
@@ -102,8 +103,18 @@ def ignore_not_applicable(text):
     return text if text.lower() != 'not applicable' else None
 
 
-def escape_spacing_in_url(url):
-    return url.replace(' ', '%20')
+def escape_url(url):
+    try:
+        rfc3987.parse(url, rule="URI")
+        return url
+    except ValueError:
+        if 'https://' in url:
+            index = 8
+        elif 'http://' in url:
+            index = 7
+        else:
+            index = 0
+        return url[:index] + quote_url(url[index:])
```

Comment:
```python
from six.moves import urllib

if url.lower().startswith('https://'):
    scheme = 'https://'
elif url.lower().startswith('http://'):
    scheme = 'http://'
else:
    scheme = ''

url = quote_url(url[len(scheme):])
return urllib.parse.urljoin(scheme, url)
```

Reply:
```urllib.parse.urljoin(scheme, url)``` produces ``http:///cds.cern.ch/record/2636102/files/Thesis%20Fiorendi.pdf`` (notice the triple bar after the scheme)
Why not just concatenate the strings? I don't think the intended use of urljoin() was this.